hack/install-deps.sh

```diff
@@ -52,19 +69,18 @@ git fetch --all
 git checkout ${RUNC_VERSION}
 BUILDTAGS=${BUILDTAGS:-seccomp apparmor}
 make BUILDTAGS="$BUILDTAGS"
-sudo make install
-which runc
```

Comment:
why remove the runc and containerd which sanity checks?

Reply:
Because now we also use `make install.deps` to install all the binaries to `release-stage`, which are not in path.

Reply:
Ah.. see it below, makes sense now.